src/main.ts

```diff
@@ -7,7 +7,7 @@ const idCanvas = 'gameCanvas'
 document.querySelector<HTMLDivElement>('#app')!.innerHTML = `<canvas id="${idCanvas}"></canvas>`
 
 window.addEventListener('load', function() {
```

Comment:
load eventlistener is now useless, remove it

Reply:
Address in 7940acb65e6c8d11f23e279dd3143786db944909